app/assets/stylesheets/generic/nav_sidebar.scss

```diff
@@ -156,18 +155,17 @@
   position: fixed;
   top: 46px;
   padding: 5px 13px 5px 13px;
-  left: 197px;
+  left: 198px;
   font-size: 13px;
-  background: #EEE;
+  background: transparent;
   color: black;
-  border-left: 1px solid rgba(0,0,0,0.035);
-  border-right: 1px solid rgba(0,0,0,0.035);
+  border-left: 1px solid $border-color;
+  border-bottom: 1px solid $border-color;
 }
 
 .collapse-nav a:hover {
   text-decoration: none;
-  color: #333;
-  background: #eaeaea;
+  background: #f2f6f7;
```

Comment:
Color literals like `#f2f6f7` should only be used in variable declarations; they should be referred to via variable everywhere else.